crates/cli/src/lib.rs

```diff
@@ -132,9 +133,32 @@ pub async fn run_cli() {
             )
             .await
         }
-        Commands::Migrate { command } => match command {
-            MigrateCommands::FromPackageJson { id } => migrate::from_package_json(id).await,
-        },
+        Commands::Migrate {
+            command,
+            skip_touched_files_check,
+        } => {
+            let maybe_workspace = load_workspace().await;
```

Comment:
does `?` not bubble up here?

Reply:
Nope it does not:
<img width="831" alt="image" src="https://user-images.githubusercontent.com/60938164/197370962-f6e3a909-ffe1-4256-83bd-aa8785141fdb.png">

Reply:
Hrmmm. There's a lot going on here. Maybe just pass the flag t the method like:

```
migrate::from_package_json(workspace, id, *skip_touched_files_check).await)
```

Reply:
You are right, I moved both the `load_workspace` and `check_dirty_repo` inside the `from_package_json` function. Looks much cleaner now.